src/openms/source/FILTERING/TRANSFORMERS/SpectraMerger.cpp

```diff
@@ -88,6 +96,8 @@ namespace OpenMS
     // same precursor MS/MS merging
     defaults_.setValue("precursor_method:mz_tolerance", 10e-5, "Max m/z distance of the precursor entries of two spectra to be merged in [Da].");
     defaults_.setMinFloat("precursor_method:mz_tolerance", 0);
+    defaults_.setValue("precursor_method:mass_tolerance", .0, "Max mass distance of the precursor entries of two spectra to be merged in [Da]. Active when set to a positive value.");
```

Comment:
this flag is never used.

Part of another PR?

Reply:
Yes it is. This is the new parameter.  If this is used, mz_tolerance is deacitivated. Probably I need put that in the explanation.

Reply:
mhh.. but the flag is new, right?
Yet I dont see it being used anywhere else in this PR...
I am probably missing something but can you point me to the location where its value is queried?